dashboard/constants.bal

```diff
@@ -42,22 +42,33 @@ const BALLERINA_BOT_TOKEN = "BALLERINA_BOT_TOKEN";
+const HEADER_EXTENDED_MODULES_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs | Load Test Results | GraalVM Check |";
+const HEADER_HANDWRITTEN_CONNECTOR_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs | Load Test Results | GraalVM Check |";
+const HEADER_GENERATED_CONNECTOR_DASHBOARD = "| Name | Latest Version | Build | Security Check | Bugs | Open PRs | GraalVM Check |";
+const HEADER_TOOLS_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs |";
 
-const LIBRARY_HEADER_SEPARATOR = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|\n";
-const HEADER_SEPARATOR = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|\n";
-const TOOLS_HEADER_SEPARATOR = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|\n";
+const HEADER_SEPARATOR_LIBRARY_MODULES = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
+const HEADER_SEPARATOR_EXTENDED_MODULES = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
+const HEADER_SEPARATOR_HANDWRITTEN_CONNECTORS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
+const HEADER_SEPARATOR_GENERATED_CONNECTORS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
+const HEADER_SEPARATOR_TOOLS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
 
-const BAL_TITLE = "### Ballerina Modules";
-const BALX_TITLE = "### Ballerina Extended Modules";
-const CONNECTOR_TITLE = "### Ballerina Connector Modules";
-const TOOLS_TITLE = "### Ballerina Tools";
+const TITLE_LIBRARY_MODULES = "### Ballerina Modules";
+const TITLE_EXTENDED_MODULES = "### Ballerina Extended Modules";
+const TITLE_HANDWRITTEN_CONNECTORS = "### Ballerina Handwritten Connector Modules";
+const TITLE_GENERATED_CONNECTORS = "### Ballerina Generated Connector Modules";
+const TITLE_TOOLS = "### Ballerina Tools";
+
+const DESCRIPTION_LIBRARY_MODULES = "These modules are published under the `ballerina` organization and packed with the Ballerina distribution.";
+const DESCRIPTION_EXTENDED_MODULES = "These modules are protocol modules that are not packed with the Ballerina distribution.";
+const DESCRIPTION_HANDWRITTEN_CONNECTORS = "These are the handwritten Ballerina connector modules that are used to connect to third-party services. They are published under the `ballerinax` organization ";
+const DESCRIPTION_GENERATED_CONNECTORS = "These are the generated Ballerina connector modules that are used to connect to third-party services. They are published under the `ballerinax` organization. The modules are generated using the Ballerina OpenAPI tool using the third-party service's OpenAPI definition. Since these are auto-generated, they only contain a smoke test suite rather than a comprehensive test suite. Due to this nature, the code coverage and load test results are not applicable for these modules.";
+const DESCRIPTION_TOOLS = "These are the Ballerina CLI tools maintained by the Ballerina team.";
```

Comment:
```suggestion
const DESCRIPTION_TOOLS = "These are the Ballerina CLI tools maintained by the Ballerina library team.";
```

Reply:
Missed this, will send a new PR.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-library/pull/6582